New package: GivEmExel v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: GivEmExel
- Repository: https://github.com/Eben60/GivEmExel.jl
- Created by: @Eben60
- Version: v0.1.0
- Commit: 8c2d11a3e1591b35656c639daf2c7909f9b8405a
- Git reference: HEAD

goerz commented:
It seems like you may have a typo in your package name. That should be `GiveEmExcel`, right?

Eben60 commented:
> That should be GiveEmExcel, right?

Then maybe `GiveThem_MSExcel®` 😁 ?

No, actually I'd prefer to leave it as it is, and save a couple of characters, if that's OK.

JuliaHubRegistrator committed:
New package: GivEmExel v0.1.0

UUID: dcf505f5-2bf4-45cb-8fef-5f56a3c0e7fc
Repo: https://github.com/Eben60/GivEmExel.jl.git
Tree: afa424189120b1e7006065ef9277489bb210b5e4

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8

goerz commented:
Sorry, I kinda screwed that up. What I meant to point out was that `GivEmExel` is missing the `c` after the `x` in `Excel`. Surely, that's a typo (or if not, I would consider that highly misleading). I'm not really concerned about `GivEm` vs `GiveEm` (or `Givem` or `GiveThem`), although me just automatically typing `GiveEm` might be an indication that people are going to be likely to have problems typing the package name.

Eben60 commented:
[noblock]
> Surely, that's a typo (or if not, I would consider that highly misleading)

That was not a typo, and I do not see how that can be misleading. It is also not about "giving them" MS® Excel®, it is about giving them a file in XLSX format (which is an ISO standard, I think). So it is about "somewhat having to to with MS Excel, or maybe not", which I think is expressed by the "incorrect" spelling. See also the following footnote from the docs:

_throughout the text, "excel file" (no capitalisation) and "XLSX file" will be used interchangeably and denote files in the XLSX format, which can be produced and read by MS Excel as well as other software, e.g. LibreOffice._

I do not expect people would really have problems typing the package name, especially as the package is not intended for the end user, but for developers of in-house packages.

Now, if you consider the current name unsuitable I'd change it exactly to whatever you propose. Thank you 🙂

goerz commented:
Huh… I'm not sure I really get it. "Exel" is not a word, just a misspelling of "Excel". I don't see how misspelling it indicates that it has "something to do with Excel, or maybe not". If you *insist*, I can unblock… It seems like a bad name to me, but if you really want that name, I don't see that I should police your decision. But maybe we should get some more maintainer eyeballs on this to see how they feel about deliberate misspellings.

`GivEmXL` or `GivEmXLSX` seem like more suitable names, if you want to avoid the Excel brand name.

[noblock]

Eben60 commented:
OK, I'm more or less OK with GivEmXL , so I'll change the package name. Thank you

goerz commented:
Closing in favor of https://github.com/JuliaRegistries/General/pull/111322
